tests/unit/niceml/experiments/test_experimentcontext.py

```diff
@@ -91,6 +92,19 @@ def test_read_write_image(experiment_context, exp_tmp_dir):
     assert np.array_equal(np_image, result_np)
 
 
+def test_write_chart(experiment_context, exp_tmp_dir):
+    """Test that it is possible to write a chart"""
+    df = pd.DataFrame(
+        {"Task": ["Task A", "Task B", "Task C", "Task D"], "Time": [10, 15, 7, 5]}
+    )
+
+    # Create a bar chart
+    chart = Chart(df).mark_bar().encode(x="Task", y="Time")
+    chart_path = "chart.html"
+    experiment_context.write_chart(chart, chart_path, False, "html")
+    assert isfile(join(experiment_context.fs_config.uri, chart_path))
```

Comment:
Not necessary, but an idea to extend the test. Save the chart as JSON and try to load the diagram from this JSON file. ([from_json](https://altair-viz.github.io/user_guide/internals.html#converting-vega-lite-to-altair))